### ISSUE TITLE
Characteristic Warning on Thermostat

### ISSUE THREAD

**Issue by dibble9012** [jvmahon/Homebridge-HomeSeer4#143]
Anyway to get this fixed so we don't have this error filling the logs (I know Homebridge will "automatically fix it", just would be nice to not have it in the first place. Thanks!

This plugin generated a warning from the characteristic 'Target Heating Cooling State': characteristic was supplied illegal value: number 4 exceeded maximum of 3. See https://git.io/JtMGR for more info.
[24/04/2021, 17:23:59] [homebridge-homeseer4] Error: 
    at TargetHeatingCoolingState.Characteristic.characteristicWarning (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2038:105)
    at TargetHeatingCoolingState.Characteristic.validateUserInput (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1950:16)
    at TargetHeatingCoolingState.Characteristic.updateValue (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1379:20)
    at Thermostat.<anonymous> (/usr/lib/node_modules/homebridge-homeseer4/lib/ThermostatSetup.js:305:7)
    at Thermostat.emit (events.js:315:20)
    at HomeSeerSystem.processReceivedData (/usr/lib/node_modules/homebridge-homeseer4/lib/HomeSeerSystemObject.js:253:15)
    at pollHomeSeer (/usr/lib/node_modules/homebridge-homeseer4/lib/Setup Listener.js:145:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

**Comment by jvmahon** [comment]
Can you post an image of the "Status Graphics" page for this thermostat and let me know what type of thermostat this is?

I don't use HomeSeer anymore, so it is a bit difficult for me to fix errors at this point, but I'll give it a try. Unfortunately, Thermostats are the most complex device types and HomeSeer does not implement them consistently between different device types.

**Comment by dibble9012** [comment]
It's a nest thermostat using the legacy nest plugin. 
[Device Settings.pdf](https://github.com/jvmahon/Homebridge-HomeSeer4/files/6528877/Device.Settings.pdf)

**Comment by jvmahon** [comment]
Unfortunately, its been a while since I looked at this code, I've switched to using Hubitat instead of HomeSeeer, and I don't have a Nest thermostat. All this combined means I don't think I can fix this anymore (not without spending a tremendous amount of time which I don't have right now). Sorry. Wish I could do more, but  Thermostats are exceedingly complex to work with (particularly if you don't have the exact model that is causing an issue). 

All this means I won't be able to help.
